setup.py

```diff
@@ -1,134 +1,23 @@
+        self.compile_assets()
+        base_sdist.run(self)
 
 
-install_requires = [
-    "Django>=4.2,<6.0",
-    "django-modelcluster>=6.2.1,<7.0",
-    "django-permissionedforms>=0.1,<1.0",
-    "django-taggit>=5.0,<7",
-    "django-treebeard>=4.5.1,<5.0",
-    "djangorestframework>=3.15.1,<4.0",
-    "django-filter>=23.3",
-    "draftjs_exporter>=2.1.5,<6.0",
-    "Pillow>=9.1.0,<12.0.0",
-    "beautifulsoup4>=4.8,<5",
-    "Willow[heif]>=1.8.0,<2",
-    "requests>=2.11.1,<3.0",
-    "openpyxl>=3.0.10,<4.0",
-    "anyascii>=0.1.5",
-    "telepath>=0.3.1,<1",
-    "laces>=0.1,<0.2",
-    "django-tasks>=0.6.1,<0.7",
-]
-
-# Testing dependencies
-testing_extras = [
-    # Required for running the tests
-    "python-dateutil>=2.7",
-    "Jinja2>=3.0,<3.2",
-    "boto3>=1.28,<2",
-    "freezegun>=0.3.8",
-    "azure-mgmt-cdn>=12.0,<13.0",
-    "azure-mgmt-frontdoor>=1.0,<1.1",
-    "django-pattern-library>=0.7",
-    # For coverage and PEP8 linting
-    "coverage>=3.7.0",
-    "doc8==0.8.1",
-    "ruff==0.9.6",
-    # For enforcing string formatting mechanism in source files
-    "semgrep==1.40.0",
-    # For templates linting
-    "curlylint==0.13.1",
-    # For template indenting
-    "djhtml==3.0.6",
-    # For validating string formats in .po translation files
-    "polib>=1.1,<2.0",
-    # For wagtail.test.utils.wagtail_factories (used for streamfield migration toolkit)
-    "factory-boy>=3.2",
-    # For running tests in parallel
-    "tblib>=2.0,<3.0",
-]
-
-# Documentation dependencies
-documentation_extras = [
-    "pyenchant>=3.1.1,<4",
-    "sphinxcontrib-spelling>=7,<8",
-    "Sphinx>=7.3",
-    "sphinx-autobuild>=0.6.0",
-    "sphinx-wagtail-theme==6.5.0",
-    "myst_parser==2.0.0",
-]
-
 setup(
     name="wagtail",
-    version=__version__,
-    description="A Django content management system.",
-    author="Wagtail core team + contributors",
-    author_email="hello@wagtail.org",  # For support queries, please see https://docs.wagtail.org/en/stable/support.html
-    url="https://wagtail.org/",
-    project_urls={
-        "Changelog": "https://github.com/wagtail/wagtail/blob/main/CHANGELOG.txt",
-        "Documentation": "https://docs.wagtail.org",
-        "Source": "https://github.com/wagtail/wagtail",
-        "Tracker": "https://github.com/wagtail/wagtail/issues",
-    },
-    packages=find_packages(),
-    include_package_data=True,
-    license="BSD",
-    long_description="Wagtail is an open source content management \
-system built on Django, with a strong community and commercial support. \
-It’s focused on user experience, and offers precise control for \
-designers and developers.\n\n\
-For more details, see https://wagtail.org, https://docs.wagtail.org and \
-https://github.com/wagtail/wagtail/.",
-    classifiers=[
-        "Development Status :: 5 - Production/Stable",
-        "Environment :: Web Environment",
-        "Intended Audience :: Developers",
-        "License :: OSI Approved :: BSD License",
```

Comment:
This was not migrated to pyproject.toml in favour of the `license` field. Keeping this in the classifier would raise a deprecation warning.